README.md

```diff
@@ -34,7 +34,7 @@
 
 # 📖 Financier <a name="about-project"></a>
 
-**Financier** is a Rails application for managing expenses and keeping track of transactions.
+**Financier** is a Rails application for managing expenses and keeping track of transactions. This allows them to save more
 
 ## 🛠 Built With <a name="built-with"></a>
```

Comment:
- [ ] For the categories page (Home page), Please add the correct placement of the `add categories` button to be at the bottom of the page to match the given design. 

https://github.com/Samuelwanza/financier/blob/f30decf7b5075dcd5ee2651363a40aefc07b633d/app/views/categories/index.html.erb#L34-L36

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/Samuelwanza/financier/assets/68796232/36d4708b-ab10-4596-8ff8-d20aeb6bc845) | ![image](https://github.com/Samuelwanza/financier/assets/68796232/c3173613-208e-4e3c-ae37-c4e3da60ffcd)  |

- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#design:~:text=Colors.-,Typography%3A%20font%20face%2C%20size%20and%20weight.,-Layout%3A%20composition%20and), Please add the correct font face (font family) for all page elements because it's supposed to be `proxima nova`, not `Nunito` as you did.

![image](https://github.com/Samuelwanza/financier/assets/68796232/91607f27-4669-47de-bf1c-9c8ed8c13f25)